static/js/components/dashboard/CourseRow.js

```diff
@@ -14,6 +19,33 @@ export default class CourseRow extends React.Component {
     now: moment$Moment,
   };
 
+  courseMiddleSection: Function = (course: Course, now: moment$Moment): React$Element<*> => {
+    let firstRun = {};
+    let price;
+    let enrollmentStatus;
+    if (course.runs.length > 0) {
+      firstRun = course.runs[0];
+      if (course.status === STATUS_OFFERED_NOT_ENROLLED || course.status === STATUS_ENROLLED_NOT_VERIFIED) {
+        if (firstRun.price) {
+          price = <span className="course-price">${firstRun.price}</span>;
+        }
+
+        if (course.status === STATUS_OFFERED_NOT_ENROLLED) {
+          enrollmentStatus = <span className="course-enrollment-description">Enrollment open</span>;
+        }
+        return (
+          <div className="course-price">
+            {price}
+            {enrollmentStatus}
+          </div>
+        );
+      }
+    }
+    return (
+      <CourseGrade course={course} now={now} />
```

Comment:
Grades aren't going to go in this column anymore. It's probably simplest to just omit this logic for now and it will get addressed in #925